english/explore-outside/journey-to-flaming-tower/jellybus/too-socially-awkward/too-socially-awkward.md

```diff
@@ -0,0 +1,9 @@
+You decide to keep walking.  You don't do well around strangers.  You're very shy.  Soon after, 
```

Comment:
No, there's a link to it.  The option was just long enough that it needed a line break.  It seems like the automated process that determines whether a file is reachable somehow breaks when the links themselves have a line break in them?  It works when I open the .md files in my browser and click the links.